src/java/org/apache/cassandra/cql3/restrictions/ClusteringColumnRestrictions.java

```diff
@@ -228,5 +228,9 @@ private boolean handleInFilter(SingleRestriction restriction, int index)
     {
         return restriction.isContains() || restriction.isLIKE() || index != restriction.getFirstColumn().position();
     }
-
+    
+    public RestrictionSet getRestrictionSet()
```

Comment:
Could this be more useful in `RestrictionSetWrapper` ?

Reply:
Makes sense...I'll pull it up.